### PR TITLE
Todo List App

### PR DESCRIPTION
**Pull request by melashu**
# In this milestone I have done the following tasks 

- I have created five separate component 
- I have used the CSS module for each component 
- I used functional component and react hook
- Users can edit previous tasks 
- User can mark the completed task as completed

### CODE REVIEW

**Review by samlorlah — APPROVED** [review]
## Hello @melashu                :wave: ,

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@samlorlah** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by samlorlah — COMMENTED** [review]
- [OPTIONAL]some of the files created by the CRA in the Public folder should still be deleted, only the Index.html file is needed.

File: src/compontes/TodoContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    setTodos((prev) => {
+      return prev.map((todo) => {
+        return todo.id === id ? { ...todo, title: title } : todo;
+      });
+    });
+  }
+
+  return (
+    <>
+      <HiMenu className="menu" />
```
Comment (samlorlah) [inline src/compontes/TodoContainer.jsx]: - [OPTIONAL] An event listener can be added to this hamburger to toggle the app menu

Comment (melashu) [reply src/compontes/TodoContainer.jsx]: Thank you so much @samlorlah

**Pull request closed without merging**